scripts/run_on_kube.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/python
+
+ECR_REPOSITORY=896129387501.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
@jkinkead separately, once I fixed my problem (of mounting the /net filesystem readonly) it successfully touched the file.  However, my job was still running after it completed.  Do I need to manually delete jobs even when they stop running?  Or are they cleaned up on some interval?